### PR TITLE
LPS-109197 SF

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java [hunk 1]
```diff
@@ -3410,6 +3410,111 @@ public void testListTemplatesWithCustomArchetypesDir() throws Exception {
...
+		Assert.assertTrue(
+			rubySassOutput,
+			rubySassOutput.contains("Using Ruby Sass compiler"));
+
+		File nativeSassOutputFile = testExists(
+			nativeSassProjectDir, "target/foo-portlet-1.0.0.jar");
+		File rubySassOutputFile = testExists(
+			rubySassProjectDir, "target/foo-portlet-1.0.0.jar");
+
+		testBundlesDiff(nativeSassOutputFile, rubySassOutputFile);
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java:3467]: @gamerson, not sure what to do about this. 

Only when comparing the maven bundles and bnd is called here: https://github.com/gamerson/liferay-portal/blob/52a28fbcc896b496efc3cc0dff8f416cff617e6f/modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/BaseProjectTemplatesTestCase.java#L897,

 It'll add this block to the errorStream

```SLF4J: Found binding in [jar:file:/opt/dev/projects/github/liferay-portal/master/.gradle/caches/modules-2/files-2.1/biz.aQute.bnd/biz.aQute.bnd/3.5.0/237e135339a5f5a2b004318b6b84e9d30f7eb52b/biz.aQute.bnd-3.5.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/opt/dev/projects/github/liferay-portal/bundles/tomcat-9.0.17/webapps/ROOT/WEB-INF/lib/util-slf4j.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/opt/dev/projects/github/liferay-portal/master/.gradle/caches/5.6.4/generated-gradle-jars/gradle-api-5.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory```

so now this test fails

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java (hunk 1 above)
Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java:3467]: weird! it only fails for me locally. i'll resolve this for now

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/hhuijser/liferay-portal/pull/4038. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**